submitted_models/ctu_cras_norlab_absolem_sensor_config_1/model.sdf

```diff
@@ -995,8 +995,36 @@
         <pose>0.13592 -0.016338 0.419596 0.015 0.499467 -0.000319</pose>
       </sensor>
     </link>
+    <joint name="laser_j" type="revolute">
+      <pose relative_to="base_link">0.2502 0 0.1407 -3.1415926535897931 0 0</pose>
+      <parent>base_link</parent>
+      <child>laser</child>
+      <axis>
+        <xyz>1 0 0</xyz>
+        <limit>
+          <lower>-2.35619</lower>
+          <upper>2.35619</upper>
+          <effort>10</effort>
+          <velocity>1.2</velocity>
+        </limit>
+        <dynamics>
+          <spring_reference>0</spring_reference>
+          <spring_stiffness>0</spring_stiffness>
+        </dynamics>
+      </axis>
+      <physics>
+        <ode>
+          <implicit_spring_damper>1</implicit_spring_damper>
+          <cfm_damping>1</cfm_damping>
+          <limit>
+            <cfm>0</cfm>
+            <erp>0.2</erp>
+          </limit>
+        </ode>
+      </physics>
+    </joint>
     <link name="laser">
-      <pose>0.2502 0 0.1407 -3.1415926535897931 -0 0</pose>
```

Comment:
`relative_to`, and frame semantics in general, is not supported in web visualization. Can you revert these changes?

Reply:
Outch... but all our models are based on the output of `ign sdf -p`, which in Ignition Dome, automatically produces these...

Reply:
Ahh. that makes sense. Please hold on this, while I consider options.

Reply:
I'd be really glad for not having to change this back :) On the other hand, as all of our robots' SDFs are generated from Xacro, you could also take a virtual machine with Blueprint, convert the files to SDF there, and upload the result to Fuel... But it sounds like it could hit us back at some point

Reply:
Updating gzweb to support 1.7 will take some effort, but it is on our plate. In the meantime, I've changed absolem to include these changes while using version 1.6 of sdf in #875. Can you take a look at the PR?

Reply:
SDF 1.6 alternative in #877.

Reply:
Is there an issue to track SDF 1.7 support in gzweb?